.travis.yml

```diff
@@ -13,7 +13,7 @@ language: ruby
 rvm:
   - '2.3.6'
```

Comment:
mind bumping these to 2.3.7 / 2.4.4 also?

Reply:
oh jeah forget those